config.json

```diff
@@ -5,5 +5,8 @@
     "enabledCommands": ["word", "hanja", "examples", "papago", "help", "msg"],
     "devIds": ["104905437614223360"],
     "llkId": "529232226546745344",
-    "enableSejongReply": true
+    "enableSejongReply": true,
+    "BALId": "819398172270264351",
+    "BATId": "888439837256994823",
+    "BAGId": "848725336366645259"
```

Comment:
moved here in case we needed them for other commands in the future, but happy to leave inside getusersinserver command